deno/generate_website_screenshot/src/mod.ts

```diff
@@ -0,0 +1,29 @@
+import { base64Encode } from "./deps.ts";
+
+const getBase64FromUrl = async (url: string) => {
+  const data = await fetch(url);
+  if (data.ok) {
+    const blob = await data.blob();
+    const arr = await blob.arrayBuffer();
+    const base64String = base64Encode(arr);
+    return { success: true, screenshot: base64String };
+  } else {
+    return { success: false, screenshot: "" };
+  }
+};
+
+export default async function (req: any, res: any) {
+  const APIkey = req.variables["SCREEN_SHOT_API_KEY"];
+  const { url } = JSON.parse(req.payload);
+  if (!url || !APIkey) {
+    res.json({ success: false, message: "Website could not be reached." });
```

Comment:
The message should be more descriptive of what was really wrong.

Reply:
it was asked in the issue.
but i will change that.

Reply:
resolved